### ISSUE TITLE
Images need alt text for accessibility

### ISSUE THREAD

**Issue by kattekrab** [openpracticelibrary/opl-media#22]
Currently our gallery images lack alt text - this mean we fail to provide basic accessibility for our site visitors.
